src/transform.ts

```diff
@@ -2,8 +2,8 @@ import * as ts from 'typescript';
 import { Collection } from './collection';
 import * as types from './types';
 
-const api = {
-  tscodeshift: (source: string|ts.Node) => {
+export const api = {
+  tscodeshift: (source: string|ts.Node): Collection<any, any> => {
```

Comment:
@KnisterPeter I had to make this change in order to get `tsc` to compile properly -- I'm guessing there's a better way to do this, but I was running into some problems locally.